subprojects/docs/src/main/resources/header.html

```diff
@@ -110,13 +111,6 @@ <h3 id="reference">Reference</h3>
             <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
         </ul>
 
-        <h3 id="getting-started">Getting Started</h3>
```

Comment:
:x: I think that significantly fewer folks will see the "Creating build scans" and "Migrating from Maven" guides since they have been demoted a ways in the organizational hierarchy.

I'd like to share an idea of how we can compromise here, and I welcome your ideas, too.

Reply:
As mentioned in a previous comment, I'm fine with the migration guide being in the left nav. I wanted it alongside the upgrade link as you have below.

The "Creating build scans" link still seems like an orphan to me that doesn't really belong at such a high level of the navigation. I would prefer to move it to the _Running Gradle builds_ section and tackle it from a more user-friendly perspective, i.e. one that users can readily understand at first look without first having build scans explained to them. I remember suggesting something like "Gaining insights into your build" and I still think we would be better off with an approach along those lines, even if a different phrase is used.